docs/sources/tempo/operations/grafana_datasource.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Grafana Tempo Datasource
+description: Use the Tempo Operator to deploy Tempo and use it as a data source with Grafana
+aliases:
+ - /docs/tempo/latest/gateway
+ - /docs/tempo/operations/gateway
+weight: 15
+---
+
+# Grafana Tempo Datasource
+
+Using the instructions on this page, you can configure the `TempoStack` to send data to Grafana and configure the Tempo data source.  
+
+You can choose to either use Tempo Operator's gateway or not: 
+
+* If the `TempoStack` is deployed using the gateway, you'll need to provide authentication information  to Grafana, along with the URL of the tenant it wants to access.
+
+* If the gateway is not used, then you need to make sure Grafana can access to the `query-frontend` endpoints.
+
+## Use with gateway
+
+The gateway, an optional component deployed as part of Tempo Operator, provides secure access to Tempo's distributor (for example, for pushing spans) and query-frontend (for example, for querying traces) via consulting an OAuth/OIDC endpoint for the request subject.
+
+The OIDC configuration expects `clientID`, `clientSecret` which should be provided via a Kubernetes secret that the `TempoStack` admin provides upfront.
```

Comment:
Is this correct that both clientID and clientSecret are provided via K8s secret or is it only clientSecret?
```suggestion
The OIDC configuration expects `clientID` and `clientSecret`. They should be provided via a Kubernetes secret that the `TempoStack` admin provides upfront.
```

Reply:
both are provided via Secret